### PR TITLE
[defakts 7541] add AI report to comment list

### PR DESCRIPTION
**Pull request by philli-m**

### CODE REVIEW

File: adhocracy4/comments/models.py [hunk 1]
```diff
@@ -39,6 +39,7 @@ class Comment(base.UserGeneratedContentModel):
     is_moderator_marked = models.BooleanField(default=False)
     # used in moderation dashboard, indicates if comment is marked as read by moderator
     is_reviewed = models.BooleanField(default=False)
+    show_ai_report = models.BooleanField(default=True)
```
[the remark this answers is not in the document]
Comment (goapunk) [reply adhocracy4/comments/models.py:42]: small note: it should be `True` as the report is shown by default

**Comment by goapunk** [comment]
@philli-m can't tag you for review here as its your PR. I think this is ready for a first round

**Comment by m4ra** [comment]
@goapunk in the dev standup @philli-m mentioned that she will cherry-pick her commit from here to her latest PR and we can close this. At least that is what i got.

**Comment by goapunk** [comment]
> @goapunk in the dev standup @philli-m mentioned that she will cherry-pick her commit from here to her latest PR and we can close this. At least that is what i got.

@m4ra she cherry-picked the a11y commit to a new branch to continue to work on that as it is unrelated to this story. I will continue to work in this PR on the story related changes.

**Review by m4ra — APPROVED** [review]

**Pull request merged**